src/dynamic-breakpoints.php

```diff
@@ -32,6 +32,9 @@ function __construct() {
 			add_action( 'rest_api_init', array( $this, 'register_settings' ) );
 
 			if ( is_frontend() && $this->has_custom_breakpoints() ) {
+				// Add a filter for replacing shortcut media queries before the breakpoint adjustment.
+				add_filter( 'stackable_frontend_css', array( $this, 'replace_shortcut_media_queries' ) );
```

Comment:
Filters need to be assigned. 

e.g.
```php
$new_value = add_filter( 'filter_name', 'old_value' );
```

Reply:
@bfintal Isn't this for `apply_filter` function? My goal here is to add a filter that replace the shortcut media queries before the breakpoint adjustment.